### PR TITLE
feat: made contact description field searchable

### PR DESCRIPTION
**Pull request by lattlay**
Makes the description field on a contact searchable.

Feat: https://github.com/monicahq/monica/issues/1819
(Note: description field already appears in contact list view)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+.idea
```
Comment (asbiin) [inline .gitignore]: What is this file ?

Comment (lattlay) [reply .gitignore]: This is the project file of the phpstorm IDE. Would you like me to remove it?

Comment (djaiss) [reply .gitignore]: I don't think we need to remove it. The .gitignore contains a bunch of stuff like this already anyway.

Comment (asbiin) [reply .gitignore]: NotI disagree. You can easily add a global gitignore for your needs. See https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Comment (lattlay) [reply .gitignore]: Cool I learned something new today! I've now removed that line from the .gitignore file

**Comment by asbiin** [comment]
Thank you @lattlay !

**Pull request merged**